release/tfregistry_test.go

```diff
@@ -178,6 +234,58 @@ func TestTFRegistryModuleReleaseLatest(t *testing.T) {
 	}
 }
 
+func TestTFRegistryProviderReleaseLatest(t *testing.T) {
+	cases := []struct {
+		client *mockTFRegistryClient
+		want   string
+		ok     bool
+	}{
+		{
+			client: &mockTFRegistryClient{
+				providerRes: &tfregistry.ProviderLatestResponse{
+					Version: "0.1.0",
+				},
+				err: nil,
+			},
+			want: "0.1.0",
+			ok:   true,
+		},
+		{
+			client: &mockTFRegistryClient{
+				moduleRes: nil,
```

Comment:
Use `providerRes` instead of` moduleRes` to clarify the intention of the test case although the default value is nil.